File: Python/Logic-1/in1to10.py
def in1to10(n, outside_mode):
  
  if outside_mode:
    if n <= 1 or n >= 10:
      return True
    else:
      return False
  elif not outside_mode:
    if n in range(1, 11):
      return True
    else:
      return False

File: Java/Logic-1/in1To10.java
public boolean in1To10(int n, boolean outsideMode) {
  
  if (n >= 1 && n <= 10 && !outsideMode 
      || n >= 10 && outsideMode
      || n <= 1 && outsideMode) 
  {
    return true;
  }
  return false;
}
